fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -3600,4 +3611,18 @@ public void testDigestAbsence() throws IOException {
             assertTrue(response.getHeaders(DIGEST).length == 0);
         }
     }
+
+    /*
+     * @see https://jira.duraspace.org/browse/FCREPO-2520
+     * @see https://jira.duraspace.org/browse/FCREPO-2658
+     */
+    @Test
+    public void Ingest_failes_with_invalid_mediatype() {
```

Comment:
Hello - can you follow the naming practice, please -- testIngestFailsWithInvalidMediatype() or something like that? 
Thanks